app.py

```diff
@@ -67,19 +67,33 @@ def delete_book(connection, book_id):
     # No Content
     return "", 204
 
+def put_book(connection, book_id):
+    body = request.json
+
+    new_book_name = body["name"]
+
+    if new_book_name== "":
+        return {"error": "Book name cannot be empty"}, 400
+
+    cursor = connection.cursor()
+    cursor.execute(f"UPDATE books SET name = '{new_book_name}' WHERE id={book_id} ")
+
+    connection.commit()
+    return {"name": new_book_name}, 201
```

Comment:
200 Status code should be returned here, 201 is "created".
https://developer.mozilla.org/en-US/docs/Web/HTTP/Status#successful_responses